### PR TITLE
add options for link sharing via mysql database

### PR DESCRIPTION
**Pull request by rekup**
- add options for link sharing via mysql database

### CODE REVIEW

File: charts/promlens/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.6 / 2021-02-08
+
+* [ENHANCMENT] Add options for link sharing via mysql database
+
```
Comment (bueti) [inline charts/promlens/CHANGELOG.md:4]: With version 1.0.4 we started to use Artifact Hub's Changelog functionality. Could move this into the Chart.yaml?

**Review by bueti — COMMENTED** [review]
Thank you for your contribution!

Looks good, just one small adjustment re the changelog.

File: charts/promlens/CHANGELOG.md (hunk 1 above)
Comment (rekup) [reply charts/promlens/CHANGELOG.md]: yes sure, 4a7c9d0b127a6cfc00da98d667ccc944c8e2517c should fix this

**Review by bueti — APPROVED** [review]
All good, thank you!

**Pull request merged**